bcipy/acquisition/protocols/lsl/lsl_client.py

```diff
@@ -74,7 +95,7 @@ def start_acquisition(self) -> bool:
 
         self.inlet = StreamInlet(
             stream_info,
-            max_buflen=self.max_buffer_len,
```

Comment:
Let's set this as our max buffer length

Reply:
It's a little confusing because there are two buffers to consider here. One is the buffer that LSL uses for the `StreamInlet`. The other is the size of our `RingBuffer` that the lsl_client uses internally. The `self.max_buffer_len` is the size used by the `RingBuffer`. These two buffers used to use the same `max_buffer_len` without issue, but for some reason that is no longer the case and we need to increase the value for the `StreamInlet` to have the data we're looking for.

The reason we need to maintain the secondary `RingBuffer` is so we can do subsequent queries for the same data. Queries to the `StreamInlet` will drain that buffer.